frontend/src/app/pages/marker-page/marker-page.component.ts

```diff
@@ -91,6 +106,13 @@ export class MarkerPageComponent implements OnInit {
                 (task: Task) => {
                     this.task = task;
 
+                    // Dev only
```

Comment:
Could you share some info about this change? why are we printing the current labelling time as an operation outside angular? 😄

Reply:
Common practice to achieve fluid clock seconds transition :P